sdks/python/setup.py

```diff
@@ -177,6 +176,7 @@ def get_version():
     'sqlalchemy>=1.3,<2.0',
     'psycopg2-binary>=2.8.5,<3.0.0',
     'testcontainers>=3.0.3,<4.0.0',
+    'avro-python3>=1.8.1,!=1.9.2,<1.10.0',
```

Comment:
We should not need to depend on this package anymore.

Reply:
Yep. Removed this from the setup.py